lively.morphic/svg-morph/svg-morph.js

```diff
@@ -162,6 +164,63 @@ export class SVGMorph extends Morph {
 
   get isSVGMorph () { return true; }
 
+  getControlPoints () {
+    this.removeAllControlPoints();
```

Comment:
just remove if there was a previous path

Reply:
if you have a look at the method you can see that I remove all ControlPoints that I can find and if I don't find any I don't remove any -> we don't get an error message